src/main/java/gmbs/model/Cars.java

```diff
@@ -0,0 +1,51 @@
+package gmbs.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import gmbs.util.RandomGenerator;
+
+import static gmbs.util.StringUtils.splitByDelimiter;
+import static gmbs.util.StringUtils.stripStringArray;
+
+public class Cars {
+    private static final String DELIMITER = ",";
+    private static final int MIN = 0;
+    private static final int MAX = 9;
+    private static final String DUPLICATED_NAME_MESSAGE = "[ERROR] 중복된 이름이 있습니다";
+    private final List<Car> cars;
+
+    public Cars(String carNames) {
+        String[] carNameArray = stripStringArray(splitByDelimiter(carNames, DELIMITER));
+        List<Car> cars = new ArrayList<>();
+        validateDuplicatedName(carNameArray);
+        for (String carName : carNameArray) {
+            cars.add(new Car(carName));
+        }
+        this.cars = cars;
```

Comment:
```suggestion
        this.cars = Arrays.stream(carNameArray)
                .map(Car::new)
                .collect(Collectors.toUnmodifiableList());
```
위와 같이 Stream을 이용해서 코드를 작성할 수 있는데요! 고려해보셔도 좋을 것 같아요~
지금 방식이 좋으시다면 유지하셔도 됩니다~